### DISCUSSION TITLE
Paginate a relationship?

### DISCUSSION THREAD

**Discussion by emogeekface** [Laravel-Backpack/community-forum#360] [Q&A (Help)]
Does anyone know if there's a way to paginate the entries from a 'Manage related entries in the same form' relationship?

I can see that I can load the entries in via ajax, but then they can't be edited from within the create/edit form of their parent, however, if there are lots of related entries then the page becomes unresponsive as it's trying to load too many things.

**Comment by pxpm** [comment]
@emogeekface can you provide a screenshot or some code so that I can see what kind of relationships/fields we are talking about ?

Cheers

**Reply by emogeekface** [reply]
@pxpm sorry for the delay, I got caught up in a go-live.

So it's a relationship fields with around 11 subfields (some of those subfields are Inline Create relationships themselves). It all functions perfectly fine, however, when you're loading in more than about 5 I think it's trying to pull in too much data/load too many relationships and it's super slow.

I have rewritten my code now to use a custom field type to display the related entries on the parent form with a button to go off and edit them and then return to the parent which makes things loads better but I can no longer manage the entries from within the parent form.

I thought a simple solution for this might be to be able to paginate or 'load more' on relationships with subfields or something.
